### ISSUE TITLE
SOF team issues.

### ISSUE THREAD

**Issue by FoxKilo** [Spearhead-Gaming-Milsim/Queens-Stallions#1]
For large testing sessions make a new issue. Individual items are preferred or grouped together to the same mod.

**Comment by FoxKilo** [comment]
Equipment boxes need to be turned 180 degrees due to accessibility issues.

Raven backpack changes back to OCP after picking it up.

SOF have no clothes.

SOF have no IFAK due to the above issue.

**Comment by FoxKilo** [comment]
SPG-9 UAZ is bugged, replicable issue of rounds hitting an invisible barrier 50m from the truck. This happens after 50% of the 12 rockets are expended. To clarify you and the enemy can fire 6 rounds from the SPG-9 gun truck and experience no problems, firing the remaining 6 rockets results in each rocket hitting a barrier approx 50m in a 360 degree radius. Occupants are safe from shrapnel except for HE rounds.

If you reload the truck to only 50%, this negates the effect.

Work-around: Load the SPG9's in EMPTY and have users load up before missions. No solution for AI vehicles which experience the same problem.

**Comment by FoxKilo** [comment]
ace_tracers not on server side. Will kick clients if loaded.

**Comment by jerkinmerkin** [comment]
Fixed uniforms and respawn markers.  Working on rest of issues.

**Comment by FoxKilo** [comment]
Loadouts need looking at too, most of the orbat is fine but I know SOF, Joint Fires, air etc are going to need some additional kit. You can either get groups to send you SQFs and do it that way or give you a list of what they'll need. #Don'tShootMessengers

**Comment by coachwn** [comment]
Already moved ace_tracers to server side for when mission restarts later tonight.

**Comment by jerkinmerkin** [comment]
Fox, please list what is missing from the loadouts

**Comment by coachwn** [comment]
Changed bug thread to specifically regard SOF slots.

**Comment by FoxKilo** [comment]
I'm going through the slots and writing up what's missing now. However just remind yourselves of what Yak said before about using SQFs for loadouts, in practicality they do make it a lot easier to setup a mission but the issue first always becomes of what the loadouts contain depending on what mods. Obviously our NATO style kits can't be used during a cherno mission so i'll find substitutes for errything.

Will update this post soon.

**Comment by FoxKilo** [comment]
Mirror all that have same names;

All VectorDay, ISR only have VectorNite
Either fill a box with lasers or add to all rifles.
suppressors for pistols.

Base Class;
Uniform;
10 packing bandage
15 elastic
3 morphine
3 epi
1 x 500ml Plasma
4 x Tourniquet

Vest;
2 x Cable Ties
2 x Strobes
XL50 maglite
map tools
Microdagr
2 x pistol mags
5+1 primary mags
2 x white smoke
2 x frags
4 x flashbangs


Whatever can't fit in vest put in backpack.

Team Leader;
Base Class + 2 x green, 2 x purple, 2 x red smoke

Comms;
Base Class + swap backpack for backpack radio. Ace SATCOM in backpack. (currently no inventory).....we can live without the SATCOM if not feasible.

Sniper;
Base Class + sniper rifle + optics + laser. bipod is not compatible. NPZ rifle does not allow switching to irons. SVDM works with this but havven't tested if it can hold accessories.

Medic;
Base Class +

Backpack;
for "_i" from 1 to 40 ACE_elasticBandage";};
for "_i" from 1 to 15 ACE_epinephrine";};
for "_i" from 1 to 15 ACE_morphine";};
for "_i" from 1 to 20 ACE_packingBandage";};
for "_i" from 1 to 3 ACE_plasmaIV";};
for "_i" from 1 to 5 ACE_plasmaIV_250";};
for "_i" from 1 to 12 ACE_tourniquet";};
surgical kit

(got really pissed and started the CTRL+C CTRL+V shenanigans)

Breacher;
Base Class + Need to ask my superior daddies what they want to do with this MOS.

Weapons;
Base Class + Need to find alternatives to fit era and capability.

ISR Team Leader;
Base Class + 

Backpack;
spotting scope + ghillie + LBV vest if current vest clips through

ISR Sniper;
Base Class + 

*will finish when I find some will to live. now ya'll know why it took me 5 hours to write loadout SQFs*

**Comment by FoxKilo** [comment]
The barrel laser attachment for das russkies doesn't work on any of the rifles except the AS Val type of rifle, which I presume we'll mostly use. Very limited number of AKs that fit with the laser to facilitate night missions. Will have to get together at a later date once the rest of mission testing is done.

**Comment by jerkinmerkin** [comment]
That's kind of the idea, AS Vals and VSS Vintorez for night fighting with lasers.  I will try to include some night vision optics in the boxes, IIRC they should be able to fit on all the rifles.

**Comment by FoxKilo** [comment]
Seems like the HLC AKs and some of the CUP ones don't accept any accessories but the RHS ones (AK103-107s and the AK-74M) mostly accept the kit. Weapons guy defo needs a GP25 variant due to the lack of substitute from mother Russia.

**Comment by jerkinmerkin** [comment]
What does weapons guy usually run with?

**Comment by jerkinmerkin** [comment]
There should be 1PN93 NVG sights in the boxes FYI

**Comment by jerkinmerkin** [comment]
Updated your loadouts and adjusted them to fit the scenario.  What you get from this point forward is what you're gonna get.

**Comment by coachwn** [comment]
Thought this was the SF team loadouts....AK-12's were not even introduced until recently and I was under impression that Jerkin time frame was 2008ish.

**Comment by coachwn** [comment]
Moved all SF team loadouts into this repo linked here...[https://github.com/Spearhead-Gaming-Milsim/SF-suggested-Russian-loadouts](url)
